.github/ISSUE_TEMPLATE/test_internal.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Test
+about: Outline the structure for a test
```

Comment:
We should make it apparent that this is for broader system/integration tests or for tests for existing code, as opposed to use this issue for each new feature.  Maybe change the name to "Planned Test" and/or change the about to something like "Outline the structure for a planned test, this could be broader system/integration tests or for tests for existing code." ?

Reply:
Thanks @KaleabTessera, makes sense. I will move some of the test questions over to the issue template then and have this one focus on planned tests.